src/routes/safe/container/hooks/useTransactionParameters.ts

```diff
@@ -81,6 +87,19 @@ export const useTransactionParameters = (props?: Props): TxParameters => {
     setEthGasPriceInGWei(toWei(ethGasPrice, 'Gwei'))
   }, [ethGasPrice, isCancelTransaction])
 
+  // Get max prio fee
+  useEffect(() => {
+    if (!ethMaxPrioFee) {
+      setEthMaxPrioFee(undefined)
```

Comment:
Why not set it to undefined when creating the state?

Reply:
But it is. The state is initialized with `undefined`

Reply:
Why is this necessary then?

Reply:
It's resetting it when the form changes. There's a ridiculous amount of abstraction layers over the form state (saved in multiple components as internal state). But we haven't touched this.